tools/functional-tester/etcd-tester/failure_agent.go

```diff
@@ -20,9 +20,15 @@ import (
 )
 
 const (
-	snapshotCount      = 10000
-	slowNetworkLatency = 500 // 500 millisecond
-	randomVariation    = 50
+	snapshotCount               = 10000
+	slowNetworkLatency          = 500   // 500 millisecond
```

Comment:
probably use `500 * time.Millisecond` (same for others) instead of having to comment about the units